client/app/services/service-details/service-details.component.js

```diff
@@ -32,6 +32,7 @@ function ComponentController ($stateParams, $state, $window, CollectionsApi, Eve
       availableTags: [],
       credential: {},
       listActions: [],
+      emptyState: {icon: 'pficon pficon-help', title: 'No data available'},
```

Comment:
Missing i18n? (since it *is* in the other `emptyState`)

(and for those `getInfo` / `provInfo` messages?)

Reply:
GOOOOOOOD catch ❤️